common/components/componentsBySection/CreateEvent/AboutEventDisplay.jsx

```diff
@@ -37,6 +37,7 @@ import EventProjectRSVPModal from "../../common/event_projects/EventProjectRSVPM
 type Props = {|
   event: ?EventData,
   viewOnly: boolean,
+  showHeaders: false
```

Comment:
This should be a boolean value unless we know for a fact it will always be false.
```suggestion
  showHeaders: boolean
```

Reply:
Okay,I have changed the showHeaders to a boolean value